src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -441,6 +441,16 @@ def transform_request(self,
         if not filters:
             filters = {"file": {}}
         filters = filters['file']
+
+        translation_dict = request_config['translation']
+        for facet in filters:
```

Comment:
@GPelayo did you test this? I think you should explicitly use `filters.keys()`.

Reply:
Both work.